lib/charms/postgresql_k8s/v0/postgresql_tls.py

```diff
@@ -54,11 +55,14 @@
 class PostgreSQLTLS(Object):
     """In this class we manage certificates relation."""
 
-    def __init__(self, charm, peer_relation):
+    def __init__(
+        self, charm, peer_relation: str, additional_dns_names: Optional[List[str]] = None
+    ):
         """Manager of PostgreSQL relation with TLS Certificates Operator."""
         super().__init__(charm, "client-relations")
         self.charm = charm
         self.peer_relation = peer_relation
+        self.additional_dns_names = [] if additional_dns_names is None else additional_dns_names
```

Comment:
```suggestion
        self.additional_dns_names = additional_dns_names or []
```

Reply:
Your suggestion is simpler and so better! I'll update the code! Thanks Mia!

Reply:
Updated on https://github.com/canonical/postgresql-k8s-operator/pull/58/commits/3c7c2dc895997bbf285cff6dc5b7de92df5cc79c.